packages/@sanity/cli/templates/shopify/README.md

```diff
@@ -206,7 +206,7 @@ npm run deploy
 npm run upgrade
 ```
 
-If you have the [Sanity CLI][sanity-cli] installed, you can also run thes with `sanity start|deploy|upgrade`. It comes with additional useful functionality.
+If you have the [Sanity CLI][sanity-cli] installed, you can also run these with `sanity start|deploy|upgrade`. It comes with additional useful functionality.
```

Comment:
Should this be changed to `\`sanity start|deploy|upgrade\``? Or is this v2?

Reply:
This template isn't actually migrated yet so I'm not quite sure what to do with it. I just came across this typo when I was doing a search for `sanity start`, so decided I might as well fix it while I was in here in case it is missed when we do migrate the template.

Reply:
Ah, then no disregard my comment 👍